### PR TITLE
Show balances for all tokens and unspents on explorer address page

### PR DESCRIPTION
**Pull request by sunify**
image

image

### CODE REVIEW

File: src/stores/explorer.ts [hunk 1]
```diff
@@ -98,14 +98,38 @@ export default class Explorer {
...
+          const token = this.tokens.tokenForColor(color);
+          if (!token) {
+            return Promise.resolve(bi(0));
+          }
+
+          return token.balanceOf(address, true);
+        })
+      )
+        .then(balances => {
+          return colors.reduce<{ [key: string]: any }>(
```
Comment (troggy) [inline src/stores/explorer.ts]: 🤯

Comment (sunify) [reply src/stores/explorer.ts]: Ah, true. I should use real type here, not any :D

**Comment by troggy** [comment]
how about making tables more dense?

Like this:
image

**Comment by sunify** [comment]
Awesome!

**Comment by sunify** [comment]
It uses existing method, so should work fine

**Pull request merged**